sdk/python/v1beta1/setup.py

```diff
@@ -22,8 +22,8 @@
     "setuptools>=21.0.0",
     "urllib3>=1.15.1",
     "kubernetes>=23.6.0",
-    "grpcio==1.41.1",
-    "protobuf==3.19.5",
+    "grpcio>=1.41.1",
+    "protobuf>=3.19.5",
```

Comment:
Thanks for updating this @tenzen-y!
Since we don't have proper tests in SDK to verify the GRPC lib functionality yet, can we update the versions in other Katib services as well. For example, [tf event collector](https://github.com/kubeflow/katib/blob/master/cmd/metricscollector/v1beta1/tfevent-metricscollector/requirements.txt#L3) and [other Suggestion services.](https://github.com/kubeflow/katib/blob/master/cmd/suggestion/hyperopt/v1beta1/requirements.txt#L7)

Reply:
You're right.
I will change the versions in this PR.

Reply:
Done.